engine/table/src/main/java/io/deephaven/engine/table/impl/sources/regioned/RegionedColumnSourceInstant.java

```diff
@@ -0,0 +1,96 @@
+package io.deephaven.engine.table.impl.sources.regioned;
+
+import io.deephaven.chunk.Chunk;
+import io.deephaven.chunk.LongChunk;
+import io.deephaven.chunk.WritableChunk;
+import io.deephaven.chunk.WritableObjectChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.table.ColumnSource;
+import io.deephaven.engine.table.impl.ColumnSourceGetDefaults;
+import io.deephaven.engine.table.impl.sources.ConvertableTimeSource;
+import io.deephaven.time.DateTime;
+import io.deephaven.time.DateTimeUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+/**
+ * Regioned column source implementation for columns of {@link Instant}s.
+ */
+final class RegionedColumnSourceInstant
```

Comment:
Can you make this class and its peers a little more mutually consistent? (`elementIndex`->`rowKey`, `final @NotNull` -> `@NotNull final`, spacing, etc)